systems/plants/RigidBody.cpp

```diff
@@ -59,8 +65,10 @@ void RigidBody::setJoint(std::unique_ptr<DrakeJoint> new_joint)
   J.resize(TWIST_SIZE, joint->getNumVelocities());
   qdot_to_v.resize(joint->getNumVelocities(), joint->getNumPositions()),
   dqdot_to_v_dqi.resize(qdot_to_v.size(), joint->getNumPositions()),
+  dqdot_to_v_dq.resize(qdot_to_v.size(), Eigen::NoChange);
```

Comment:
so there are two `setup calls` for a RigidBody: `setN` and `setJoint`. The number of joint positions and velocities is known when `setJoint` is called, whereas the size of the robot's q vector is known when `setN` is called. I need to know both sizes to resize dqdot_to_v_dq, so I resize the rows only in `setJoint` and the columns only in `setN`.